[2023-07-16 02:23:51,642] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_api_marvel_hacia_aws_redshift_multitask.extraer_data manual__2023-07-16T02:23:49.944661+00:00 [queued]>
[2023-07-16 02:23:51,658] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl_api_marvel_hacia_aws_redshift_multitask.extraer_data manual__2023-07-16T02:23:49.944661+00:00 [queued]>
[2023-07-16 02:23:51,658] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-16 02:23:51,658] {taskinstance.py:1377} INFO - Starting attempt 1 of 11
[2023-07-16 02:23:51,659] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-16 02:23:51,675] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extraer_data> on 2023-07-16 02:23:49.944661+00:00
[2023-07-16 02:23:51,682] {standard_task_runner.py:52} INFO - Started process 9455 to run task
[2023-07-16 02:23:51,685] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_api_marvel_hacia_aws_redshift_multitask', 'extraer_data', 'manual__2023-07-16T02:23:49.944661+00:00', '--job-id', '862', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_multiple_tasks.py', '--cfg-path', '/tmp/tmp12rcdnri', '--error-file', '/tmp/tmp9gl2g1bc']
[2023-07-16 02:23:51,686] {standard_task_runner.py:80} INFO - Job 862: Subtask extraer_data
[2023-07-16 02:23:51,758] {task_command.py:371} INFO - Running <TaskInstance: etl_api_marvel_hacia_aws_redshift_multitask.extraer_data manual__2023-07-16T02:23:49.944661+00:00 [running]> on host 7a1b6d75866e
[2023-07-16 02:23:51,857] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=arielmfeldman.af@gmail.com
AIRFLOW_CTX_DAG_OWNER=Ariel Feldman
AIRFLOW_CTX_DAG_ID=etl_api_marvel_hacia_aws_redshift_multitask
AIRFLOW_CTX_TASK_ID=extraer_data
AIRFLOW_CTX_EXECUTION_DATE=2023-07-16T02:23:49.944661+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-16T02:23:49.944661+00:00
[2023-07-16 02:23:53,273] {logging_mixin.py:115} INFO - ya existe el archivo
[2023-07-16 02:23:53,290] {python.py:173} INFO - Done. Returned value was: None
[2023-07-16 02:23:53,305] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=etl_api_marvel_hacia_aws_redshift_multitask, task_id=extraer_data, execution_date=20230716T022349, start_date=20230716T022351, end_date=20230716T022353
[2023-07-16 02:23:53,343] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-16 02:23:53,391] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
